<?xml version="1.0" encoding="utf-8"?>
<launch>
  <group ns="camera/rgb">
    <node pkg="chessboard_pose" type="chessboard_pose" respawn="false" output="screen" name="chessboard_detector">
      <param name="display" value="1"/>
      <param name="frame_id" value=" /camera_rgb_optical_frame"/>
      <param name="rect_x_size" value="0.02"/>
      <param name="rect_y_size" value="0.02"/>
      <param name="grid_x_size" value="9"/>
      <param name="grid_y_size" value="6"/>
      <param name="publish_tf" value="true"/>
    </node>
  </group>


<node pkg="chessboard_publisher" type="publisher" name="chessboard_4_3" output="screen">
  <param name="name" value="chessboard" />
  <param name="grid_x_size" value="10" />
  <param name="grid_y_size" value="7" />
  <param name="rect_x_size" value="0.02" />
  <param name="rect_y_size" value="0.02" />
</node>

</launch>
